app/controllers/staffs_controller.rb

```diff
@@ -0,0 +1,39 @@
+class StaffsController < ApplicationController
+
+  before_action :authenticate_customer!
+  
+  def index
+    @staffs = Staff.all(_account_id: current_customer.current_account_id).fetch
```

Comment:
@deepali1-aconnexion does it possible to add belongs_to relation to Staff model and use it like `current_customer.current_account.staff.build` here?